src/raft_server.c

```diff
@@ -204,12 +203,14 @@ void raft_become_follower(raft_server_t* me_)
 int raft_periodic(raft_server_t* me_, int msec_since_last_period)
 {
     raft_server_private_t* me = (raft_server_private_t*)me_;
+    raft_node_t *my_node = raft_get_my_node((void *)me);
 
+    assert(NULL != my_node);
```

Comment:
When a new replica starts with empty membership, we will fail this assertion, won't we?

Reply:
If it is because of the way the code is written, then we should change the code to accept requests from unknown replicas. The guiding principle in Raft is "servers process incoming RPC requests without consulting their current configurations", I think.